R/df2bib.R

```diff
@@ -52,7 +52,7 @@ df2bib <- function(x, file = "", append = FALSE) {
       if (is.list(f)) {
         f <- unlist(f)
       }
-      rowfields[[i]] <- if (!length(f) || is.na(f)) {
+      rowfields[[i]] <- if (!length(f) | is.na(f)[1]) {
```

Comment:
In this case the naming is much worse. I'd guess from context that `f` is short for `field` since the list returned by the `lapply()` call is eventually stored as `fields`.  If that's the case, I'd expect `length(f) == 1` and would vote for renaming `f` to `field` at which point my comment above concerning `file` would apply here as well.

With just a 'mystery `f`' about we know nothing but `length(f) >= 1`, I'd actullay prefer

```suggestion
      rowfields[[i]] <- if (!length(f) | any(is.na(f))) {
```

as it does not force me to think about what `f[1]` means. Also, if `length(f) > 1` was a vaild scenario `is.na(f[1])` seems more straightforward that `is.na(f)[1]` to me.

Reply:
`f` refers to elements within the field. For instance `f` is a list and has `length(f) > 1` whenever there is more than one author in the author field of a given `.bib` entry. Again, I completely agree with your suggestion and the reason I implemented mine was to stick as close as possible to the original code for fear of breaking other things :)